src/style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./Avenir-Book.ttf);
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search_input {
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: inherit;
+}
+
+.search_input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search_input:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.search_input::placeholder {
+  font-weight: normal;
+}
+
+.big_div {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.small_div {
+  height: 42px;
+}
+
+.search_icon {
+  position: absolute;
```

Comment:
don't separate position from other styles of this element, it's ok to repeat position property